modules/logging-release-notes-5-9-10.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging_release_notes/logging-5-9-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-9-10_{context}"]
+= Logging 5.9.10
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:????[????].
+//OpenShift Logging Bug Fix Release 5.9.10].
+
+[id="logging-release-notes-5-9-10-bug-fixes_{context}"]
+== Bug Fixes
```

Comment:
Add https://issues.redhat.com/browse/LOG-6376 as requested in https://github.com/openshift/openshift-docs/pull/86331#discussion_r1887637904.

Reply:
```
Before this update, when tls.insecureSkipVerify: true was defined alongside type: http and not secure URL in the Cluster Log Forwarder configuration, the certificate validation was not skipped as intended. This misconfiguration led to the collector failing because it attempted to validate certificates despite the setting.

With this update, we've added additional validation for the URL scheme. Now, if tls.insecureSkipVerify: true is set but the URL is not secure (i.e., does not use HTTPS), a misconfiguration error is raised, thus preventing potential security issues and ensuring the configuration behaves as expected.
```